cmd/run.go

```diff
@@ -769,4 +793,13 @@ func bindRunFlags(cmd *cobra.Command) {
 
 	cmd.Flags().Uint32("listObjects-max-results", defaultConfig.ListObjectsMaxResults, "the maximum results to return in ListObjects responses")
 	util.MustBindPFlag("listObjectsMaxResults", cmd.Flags().Lookup("listObjects-max-results"))
+
+	cmd.Flags().Bool("metrics", defaultConfig.Metrics.Enabled, "toggle OTel metrics exporter")
```

Comment:
`metrics-enabled` 

to be consistent with the other flags (e.g `--playground-enabled`, `--http-enabled`, etc..)